tutorials/raw/40_visualize_raw.py

```diff
@@ -114,12 +114,25 @@
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
 # To visualize the frequency content of continuous data, the `~mne.io.Raw`
-# object provides a `~mne.io.Raw.plot_psd` to plot the `spectral density`_ of
-# the data.
+# object provides a :meth:`~mne.io.Raw.compute_psd` method to compute
+# `spectral density`_ and the resulting :class:`~mne.time_frequency.Spectrum`
+# object has a :meth:`~mne.time_frequency.Spectrum.plot` method:
 
-raw.plot_psd(average=True)
+spectrum = raw.compute_psd()
+spectrum.plot(average=True)
 
 # %%
+# .. note::
+#
+#    Prior to the addition of the :class:`~mne.time_frequency.Spectrum` class,
+#    the above plot was possible via::
+#
+#        raw.plot_psd(average=True)
+#
+#    The :meth:`~mne.io.Raw.plot_psd` method of :class:`~mne.io.Raw` objects
+#    is still provided to support legacy analysis scripts, but new code should
+#    instead use the :class:`~mne.time_frequency.Spectrum` object API.
```

Comment:
This one by contrast seems fine because the old API is *not* used immediately afterward